lib/interpreters/split_clustering.rb

```diff
@@ -793,13 +793,19 @@ def self.collect_cluster_data(vrp, nb_clusters)
     def self.duplicate_vehicle(vehicle, timewindow, schedule)
       if timewindow.nil?
         (schedule[:start]..[schedule[:end], 6].min).collect{ |day|
+          vehicle.skills = vehicle.skills.collect{ |sk_set|
+            sk_set | [%w[mon tue wed thu fri sat sun][day]]
```

Comment:
Can we have these days in symbol `%i[]` if the rest of the change is not that painful? 

With string, we allocate memory for each comparison `(v_skills - s_skills).empty?` even inside the gem. For "words" that are general enough symbol is better than string because memory is allocated once.

Reply:
On top of this, we can extract the `%i\w[mon tue wed thu fri sat sun]` to a class constant such as `WEEKDAYS = %i\w[mon tue wed thu fri sat sun].freeze` and use it whenever necessary, instead of creating a new array each time we need such a comparison.

Reply:
For the sake of efficiency I suggest that I do these changes at the same time as I work on https://gitlab.com/mapotempo/optimizer-api/-/issues/635 (which I am actually working on right now). This will be great to differenciate internal and original skills.

Reply:
In that dev, you can even turn general skills into symbol at the API level (since form the user point of view nothing changes -- string and symbol are the in JSON). We know that we cannot have a VRP with an infinite number of different skills. That change would make skill comparisons faster as well.